web3-dev/useful-resources.rst

```diff
@@ -0,0 +1,20 @@
+.. index:: library, useful resources
+
+.. _web3-useful-resources:
+
+################
+Useful resources
+################
+
+Here is a list of useful resources for smart contract development.
+
+- `massa-web3 <https://github.com/massalabs/massa-web3>`_ is a TypeScript library
+  that allow you to interact with the Massa blockchain through a local or remote Massa node.
+- `create-react-app-massa <https://github.com/massalabs/create-react-app-massa>`_ is a minimal
+  React template made for our massa-web3 library.
+- `massa-sc-examples <https://github.com/massalabs/massa-web3>`_ is a collection of
```

Comment:
same here do you mean https://github.com/massalabs/massa-sc-examples/tree/main/blog?

Reply:
Not sure what you refer to ?

Reply:
The line explain `massa-sc-examples` but have a link to `massa-web3` @qdrn

Reply:
Thanks. Did not notice it was on another file. Corrected.